ambry-router/src/main/java/com.github.ambry.router/GetBlobOperation.java

```diff
@@ -830,7 +834,7 @@ void postCompletionCleanup() {
 
     @Override
     GetOptions getGetOptions() {
-      return GetOptions.None;
+      return getOptions;
```

Comment:
Could we just call `options.getGetOptions()` here instead of adding a new variable?